ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -139,7 +140,10 @@ + (NSURLRequest *)canonicalRequestForRequest:(NSURLRequest *)request
 {
     AD_LOG_VERBOSE_F(@"+[ADURLProtocol canonicalRequestForRequest:]", _reqCorId(request), @"host: %@", [request.URL host] );
     
-    return request;
+    NSMutableURLRequest* mutableRequest = [request mutableCopy];
```

Comment:
As we tested it, the 60s timeout can also come out from the request created by webKit. So canonicalRequestForRequest is the only place where we can modify that request :)